content/docs/authorization/release/_index.md

```diff
@@ -26,5 +23,6 @@ Description: >
 
 
 ### Known Issues
-
-There are no known issues in this release.
+| Issue | Workaround |
+|-------|------------|
+| CSM Operator does not support dynamic namespaces for Authorization. Despite successful installation in a namespace other than "authorization", errors may arise during volume creation. | Use the default namespace "authorization" for installing Authorization using CSM Operator|
```

Comment:
```suggestion
| Issue | Workaround |
|-------|------------|
```
No need to add this as it will be fixed in CSM 1.11

Reply:
The changes have been made.